### ISSUE TITLE
Tutorial to be more explicit about when something is supposed to be working

### ISSUE THREAD

**Issue by thoni56** [Urigo/Ionic2-MeteorCLI-WhatsApp#28]
I run `meteor` continuously as I walk through the tutorial, but it is not clear to me when I should expect to see something working.

It would be very helpful if the tutorial said something along the lines of "at this point you should see/be able to..." at points where such a landmark is reached.

**Comment by NicholasHattabaugh** [comment]
I agree as i still don't get anything to appear in the browser, i've been trying to get this tutorial to work for the past few days and so far nothing, is there something else you need to install or do to get this to work?
